simplemonitor/Alerters/healthchecks.py

```diff
@@ -0,0 +1,127 @@
+"""
+SimpleMonitor alerts via healthchecks
+"""
+
+from typing import cast
+
+import json
+import requests
+
+from ..Monitors.monitor import Monitor
+from .alerter import Alerter, AlertLength, AlertType, register
+from ..util import check_group_match
+
+
+@register
+class HealthchecksAlerter(Alerter):
+    """Send push notification via Healthchecks."""
+
+    alerter_type = "healthchecks"
+    default_headers = {"User-Agent": "SimpleMonitor"}
+
+    def __init__(self, config_options: dict) -> None:
+        super().__init__(config_options)
+        self.hc_token = cast(
+            str, self.get_config_option("token", required=True, allow_empty=False)
+        )
+        self.hc_create = cast(str, self.get_config_option("create", required_type=bool))
+        hc_headers = cast(str, self.get_config_option("headers"))
+        if hc_headers:
+            try:
+                self.hc_headers = json.loads(hc_headers)
```

Comment:
Worth doing a union kind of thing here so that the value(s) in the config merge/overwrite the default? I'm thinking about the User-Agent (like the idea of setting that) being lost if it's not included by the user's configuration. Merging the two would mean it's there if not otherwise set, and the user can override it if they want.

Reply:
Do you mean to get the defaults.headers value from monitors.ini first as the default for the healthcheck headers ?

I think this shall not be used in the Alerters context. The monitor headers are meant to be used for checking requests - the alerter headers for reporting. So these are different things. 

It might be useful though to have defaults section for Alerters (and maybe Loggers) - same as for the monitors - ie.:

config/monitor.ini
config/monitors/monitors.ini
config/alerters/alerters.ini
...

But this goes beyond this PR.

Reply:
I just meant specifically here that you're defining the default headers with the useragent value, but if the user sets some in their ini file the useragent entry is lost by the assignment at line 31 (unless the user also includes it) :)

I was thinking like `self.hc_headers.update(json.load(hc_headers))` or something

Reply:
On future config work, I agree that would be useful :) This project very much grew organically as I needed features for the use case I made it for originally (also I was learning Python at the time), which is why there's some oddities in there!

Reply:
I have cleaned up the class a bit.

The desired behaviour is if user does not set headers in the config the default header "{"User-Agent": "SimpleMonitor"}" shall be used.

Reply:
https://github.com/jamesoff/simplemonitor/pull/1464/commits/80c3ec5ae7bf0cdbef796fbd0f4eb8a3a76b4eca#diff-b8842d3da10f9b787cd6ca6d3e2ec36f8bda7871850de39ef29d25d6c73f109e